deepsensor/model/model.py

```diff
@@ -648,6 +648,597 @@ def unnormalise_pred_array(arr, **kwargs):
 
         return pred
 
+    def predict_patch(
```

Comment:
'predict_patch' makes me think we will predict one single patch, rather than stitch predictions over many patches together into one prediction. How about 'predict_patchwise'?

Reply:
[527edff](https://github.com/alan-turing-institute/deepsensor/pull/135/commits/527edff5162ddee5cee786ace5d40532170f9fb9)